google/cloud/spanner/ci/run_integration_tests_emulator_cmake.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
```

Comment:
I suggest adding

```
source "$(dirname "$0")/../../../../ci/lib/init.sh"
source module lib/io.sh
```

Then you can get rid of `BINARY_DIR` in favor of `PROGRAM_DIR`, and `PROJECT_ROOT` will already be set for you. You can also use `io::log` if you want (or don't log and then you don't need to source it).

Reply:
Done.  As you point out below, `BINARY_DIR` is really the CMake binary directory, renamed the variable and added a comment.